modules/eap/module.yaml

```diff
@@ -13,7 +13,7 @@ modules:
           # you first verified the image with changed order still works correctly
           # Common modules from the main CE cct_module repository
           - name: dynamic-resources
-          - name: jboss.container.eap.s2i.bash
+          - name: jboss.container.eap.s2i
```

Comment:
@drichtarik Keep the file you changed as is.

But replace `jboss.container.maven.s2i.bash` with `jboss.container.maven` in these two files:

- `modules/jboss/container/eap/s2i/bash/module.yaml:18:  - name: jboss.container.maven.s2i.bash`
- `modules/jboss/container/eap/s2i/bash/README.adoc:54:link:../../../../../jboss/container/maven/s2i/bash/README.adoc[jboss.container.maven.s2i.bash]`

Reply:
Why replace `jboss.container.maven.s2i.bash` with `jboss.container.maven` and not `jboss.container.maven.s2i`?